python/taichi/lang/matrix.py

```diff
@@ -26,9 +26,9 @@ class Matrix(TaichiOperations):
     """The matrix class.
 
     Args:
-        n (int): the first dimension of a matrix.
+        n (int, list, tuple, np.ndarray): the first dimension of a matrix.
```

Comment:
nit: `n (Union[int, list, tuple], np.ndarray)`

Reply:
@ailzhang I have improve it.